ops/services/virtual_network/main.tf

```diff
@@ -25,11 +25,11 @@ resource "azurerm_subnet" "vms" {
   # We did try to rename this, but it turns out we have manual infrastructure attached to this subnet,
   # so renaming it (by which we mean destroying and rebuilding it) would be a bit complicated.
   # That manual infrastructure should be imported (see #1360)--if that is done, revisit this.
-  name                                           = "${var.env}-vms"
-  resource_group_name                            = var.resource_group_name
-  virtual_network_name                           = azurerm_virtual_network.vn.name
-  address_prefixes                               = [cidrsubnet(var.network_address, 8, 252)] # X.X.252.0/24
-  enforce_private_link_endpoint_network_policies = true
+  name                                      = "${var.env}-vms"
+  resource_group_name                       = var.resource_group_name
+  virtual_network_name                      = azurerm_virtual_network.vn.name
+  address_prefixes                          = [cidrsubnet(var.network_address, 8, 252)] # X.X.252.0/24
+  private_endpoint_network_policies_enabled = false
```

Comment:
Whats the reason for this change? why is this being turned off?

Reply:
This was updated to resolve zdeprecations, we're moving to `azurerm` 4 soon so it needs to happen. 

I've pushed a revision with this set to true, I'll run a plan to if it was mistakenly set to false or if I did it for a reason. (I don't remember) This was set a while back and the tf plans have looked good so it feels like it should be false but I understand the concern.

Reply:
So, setting it to `true` ends up updating the resource, which is something I try to avoid in these types of updates. I will remove this change entirely because this is a subnet for VMs; VM resources will soon be deleted, and I don't want to waste time on this when I know I'll just be removing it soon anyways.